### ISSUE TITLE
problem: no consistency in page headers/labels

### ISSUE THREAD

**Issue by AshHimself** [Blockrazor/blockrazor#1294]
Problem: no consistency in page headers/labels and are different on each page. e.g. centered, aligned left.

Solution: Discuss and agree on a standard and then apply globally.

**Comment by gazhayes** [comment]
This should probably be somehow global in a truly global sense so that even if we decided to change it it only has to be changed in one place.

**Comment by AshHimself** [comment]
Correct. e.g. all headers should be in <div class='brazor-header></div> or
whatever.

**Comment by AshHimself** [comment]
@blockrazorbot claim

**Comment by gazhayes** [comment]
> So if you are ok with it we can roll this out to every single page and have a little more flow and UX.

I haven't looked but I assumed that's what you were going to do anyway so don't let me hold you back.

> The panel is actually good to add breadcrumbs too if you get feedback of people getting lost where they are. I do know flowrouter has some sort breadcrumbs package.

I think you (or someone) logged that problem and the solution was to use an active class for menu items?

**Comment by AshHimself** [comment]
@gazhayes i've added it to all pages now which have been styled. We can gradually add the new style to all other pages as we bootwatchify them. #1304

**Comment by AshHimself** [comment]
@blockrazorbot label "done"

**Comment by AshHimself** [comment]
@blockrazorbot done
